test/test_file_processor.py

```diff
@@ -72,6 +83,54 @@ def test_load_csv_with_cp932(self):
                 pd.testing.assert_frame_equal(df, loaded_df)
 
 
+class TestJsonFileProcessor:
+    @pytest.mark.parametrize(
+        'orient,input_data,expected_json',
+        [
+            pytest.param(
+                None,
+                pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]}),
+                '{"A":{"0":1,"1":2,"2":3},"B":{"0":4,"1":5,"2":6}}',
+                id='With Default Orient for DataFrame',
+            ),
+            pytest.param(
+                'records',
+                pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]}),
+                '{"A":1,"B":4},{"A":2,"B":5},{"A":3,"B":6}',
```

Comment:
It's more clear to use raw result in tests, because tests indicates how this feature works.
This tests includes transformations about the results of this function, which makes the expected feature ambiguous.
So please omit the following code, and use raw value in test parameters.

```python
                 # newline delimited json file
                  _loaded_json = [line.decode('utf-8').strip() for line in f.readlines()]
                  loaded_json = ','.join(_loaded_json)
```
```suggestion
                '{"A":1,"B":4}\n{"A":2,"B":5}\n{"A":3,"B":6}',
```

Reply:
Hi, I have modified this part in the commit below to verify the original JSON result.
https://github.com/m3dev/gokart/pull/458/commits/17444230d4981d3731f8849272a3ebae88c06893